src/main/java/chess/controller/WebController.java

```diff
@@ -0,0 +1,71 @@
+package chess.controller;
+
+import chess.dto.BoardDto;
+import chess.dto.GameDto;
+import chess.dto.MoveDto;
+import chess.dto.StatusDto;
+import chess.service.ChessService;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class WebController {
+
+    private final ChessService chessService;
+
+    @Autowired
+    public WebController(ChessService chessService) {
+        this.chessService = chessService;
+    }
+
+    @GetMapping("/")
+    public ModelAndView selectGame() {
+        BoardDto boardDto = chessService.selectBoard();
+        String winner = chessService.selectWinner();
+
+        Map<String, Object> model = new HashMap<>();
+        model.put("board", boardDto);
+        model.put("winner", winner);
+
+        return new ModelAndView("index", model);
+    }
+
+    @PostMapping("/game")
+    @ResponseBody
+    public ResponseEntity insertGame() {
```

Comment:
```suggestion
    public ResponseEntity<GameDto> insertGame() {
```

제네릭을 명시하지 않으면 어떠한 문제가 있을지 고민해보아요